### PR TITLE
fix: pcdLoader parse header multiple types

### PR DESCRIPTION
**Pull request by xiaomo1q**

### CODE REVIEW

**Comment by ibgreen** [comment]
@xiaomo1q  thanks for the contribution. If you'd like this to be considered for merging:
-  First you need to run `yarn lint fix` to fix the linter/prettier issues (or perhaps just manually remove the extra spaces in the import statements.
- Also can you provide some context in the PR description
- Add some comment to the code
- Remove the commented out code

**Comment by xiaomo1q** [comment]
the pcdHeader likely contains multiple data points for each field
`      pcdHeader.offset[pcdHeader.fields[i]] = sizeSum
        sizeSum += pcdHeader.size[i] * pcdHeader.count[i]; ` 
By multiplying the size of each field by its count, calculates the total size occupied by that field in the data buffer.

File: modules/pcd/src/lib/parse-pcd.ts [hunk 1]
```diff
@@ -6,11 +6,10 @@
...
 
-import {MeshAttribute, MeshAttributes} from '@loaders.gl/schema';
-import {getMeshBoundingBox} from '@loaders.gl/schema';
-import {decompressLZF} from './decompress-lzf';
-import {getPCDSchema} from './get-pcd-schema';
-import type {PCDHeader, PCDMesh} from './pcd-types';
+import { getMeshBoundingBox, MeshAttribute, MeshAttributes } from '@loaders.gl/schema';
+import { decompressLZF } from './decompress-lzf';
+import { getPCDSchema } from './get-pcd-schema';
+import type { PCDHeader, PCDMesh } from './pcd-types';
```
Comment (ibgreen) [inline modules/pcd/src/lib/parse-pcd.ts:12]: The spaces you added are causing the lint to fail. Regardless of your personal preferences, when you are contributing to another repo, you need to adopt the conventions in that repo:
```suggestion
import {getMeshBoundingBox, MeshAttribute, MeshAttributes} from '@loaders.gl/schema';
import {decompressLZF} from './decompress-lzf';
import {getPCDSchema} from './get-pcd-schema';
import type {PCDHeader, PCDMesh} from './pcd-types';
```

**Comment by xiaomo1q** [comment]
Adopting Lint format

**Comment by ibgreen** [comment]
@xiaomo1q Closing due to inactivity. Please reopen if you can fix the lint issues and add a test case that leverages the new code.

**Pull request closed without merging**